test/distributed/_tensor/test_math_ops.py

```diff
@@ -16,29 +16,47 @@
 
 
 class DistMathOpsTest(DTensorTestBase):
-    @with_comms
-    def test_sum(self):
-        device_mesh = self.build_device_mesh()
 
+    def linear_op_reductions(self, op_str):
+        device_mesh = self.build_device_mesh()
         shard_spec = [Shard(0)]
 
-        tensor_to_sum = torch.randn(12, 8, 8)
+        x = torch.randn(12, 8, 8)
+        x_dt = distribute_tensor(x, device_mesh, shard_spec)
```

Comment:
It's a small thing but `x_dt` looks a bit too much like a differential equation.

Characters are _free_, so can we call `x`->`tensor` and `x_dt`->`distributed_tensor`? The clearer the tests the better.